resources/views/layouts/app.blade.php

```diff
@@ -99,7 +97,11 @@
                             </div>
                         </li>
 
-
+                        <script>
```

Comment:
what's the purpose of this code here? Seems not doing anything for the page.

Reply:
I didn't this code on the page.... maybe did someone leave?

Reply:
If it's not being used, maybe we can remove this part of code